### ISSUE TITLE
Add a `<Switch>` to non-repeating `<Route>`s in App.js

### ISSUE THREAD

**Issue by knod** [codeforboston/cliff-effects#682]
Some `<Route>`s need to be on every page (like the one for the header). Those ones shouldn't be in the `<Switch>`. Others cancel each other out - if we're on the About page, none of the form stuff matters.

We need to put a `<Switch>` in the appropriate place and maybe switch up which order the routes appear in so that we can always get to the right place (one 'load' may need to be in a different order).

**Comment by ethanbb** [comment]
Yeah this makes sense to me. Although it works the way it is now, semantically having a `<Switch>` for the main content makes sense, and I think it could enable more flexibility for routing in the future.
